Assets/Scripts/Editor/ResetTransforms.cs

```diff
@@ -0,0 +1,17 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace UnityLibrary
+{
+    public static class ResetTransforms
+    {
+        [MenuItem("GameObject/Reset Transform #r")]
+        public static void ResetTransform()
+        {
+            GameObject go = Selection.activeGameObject;
+            if (go == null) return;
+            Undo.RegisterCompleteObjectUndo(go.transform, "Transform Reset");
+            go.transform.SetPositionAndRotation(Vector3.zero, Quaternion.identity);
```

Comment:
No scale reset?

Reply:
No, this is just to put an object at 0,0,0 when you create a new object